### PR TITLE
[Fixes #4099] Add a functional test for creating a form with a file upload field

### LINKED ISSUE #4099: create a form with upload a photo
How can I create a basic model (username, password etc...) and with a photo and submit it to the controller? Please, example...

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

File: test/WebSites/FilesWebSite/Models/User.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        public int Age { get; set; }
+        public IFormFile Biography { get; set; }
+
+        public string ReadBiography()
+        {
+            if (Biography != null)
+            {
+                using (var reader = new StreamReader(Biography.OpenReadStream()))
+                {
+                    return reader.ReadToEnd();
```
Comment (kichalla) [inline test/WebSites/FilesWebSite/Models/User.cs]: nit: ReadToEndAsync ?

**Comment by kichalla** [comment]
:shipit:

File: test/WebSites/FilesWebSite/Models/User.cs (hunk 1 above)
Comment (javiercn) [inline test/WebSites/FilesWebSite/Models/User.cs]: No need to complicate things

Comment (kichalla) [inline test/WebSites/FilesWebSite/Models/User.cs]: I agree that this is just a test, but if our plan is to show this as a sample to our users, I think this should be `async`

Comment (javiercn) [inline test/WebSites/FilesWebSite/Models/User.cs]: fair enough :)

**Comment by ajaybhargavb** [comment]
:shipit:

File: test/WebSites/FilesWebSite/Models/User.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace FilesWebSite.Models
+{
+    public class User
+    {
+        public string Name { get; set; }
+        public int Age { get; set; }
+        public IFormFile Biography { get; set; }
+
+        public async Task<string> ReadBiography()
```
Comment (kichalla) [inline test/WebSites/FilesWebSite/Models/User.cs]: nit: `ReadBiographyAsync`

**Pull request merged**